test/index.js

```diff
@@ -18,6 +18,9 @@ const request = require('supertest');
 const should = require('should');
 const Koa = require('koa');
 const Router = require('koa-router');
+const sinon = require('sinon');
```

Comment:
I had to bring in `sinon` to the `devDependencies` to hook into the test server request handlers. Symbols for raw bodies are applied serverside on `ctx.request`, and since symbols by definition wont be transferred over the network, I couldn't test with the existing setup.